src/Nox.Ui.Blazor.Lib/Components/NoxTypes/ViewEntityExactlyOne.razor.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class ViewEntityExactlyOne : ComponentBase
```

Comment:
Let's discuss if we need this component. I guess for zeroToMany relationship we can display the link to the selected related entity or at least its name. Don't think we need a separate component for it.

Reply:
ok I agree I have removed it - and we can just use SelectEntityModel.Name for display